docs/labwc-config.5.scd

```diff
@@ -208,6 +208,9 @@ this is for compatibility with Openbox.
 
 		- *type* Show view type ("xdg-shell" or "xwayland")
 
+		- *winfo* Show workspace name, *type* "X" or "W"
```

Comment:
I prefer not to do a compound type like this. It feels to 'hardcoded' to suit one particular preference, but with not flexibility.

Consider implementing `type_short`, `state` and `workspace` as separate items.

Consider a `custom` type where we could support fields in a string, for example "%w %s %t" or something like that. That way we keep the flexibility in terms of separators, spaces, etc.